### PR TITLE
Background phase transition

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

File: src/Persistent/Idiom.hs [hunk 1]
```diff
@@ -157,13 +156,8 @@ saveAndEnactFreeze :: Timestamp -> Freeze -> AUpdate ()
...
   dbFreeze .= shouldBeFrozenOrNot
   let change = case shouldBeFrozenOrNot of
-          NotFrozen -> PhaseThaw   now
-          Frozen    -> PhaseFreeze now
-      freezeOrNot topic = do
-          case phaseTrans (topic ^. topicPhase) change of
-            Nothing -> return ()  -- ignored, probably repeated freeze or thaw
-            Just (phase', actions) -> do
-                let !_A = assert (null actions) ()  -- wrong monad to execute'em
-                setTopicPhase (topic ^. _Id) phase'
```
Comment (fisx) [inline src/Persistent/Idiom.hs]: so, the assertion is now eliminated?  i think i'll try to re-add it, hope that's ok?

Comment (fisx) [inline src/Persistent/Idiom.hs]: hum, you have very thoroughly removed that option, it seems.

Comment (np) [inline src/Persistent/Idiom.hs]: yes the assertion is no longer necessary

**Comment by fisx** [comment]
LGTM!  i reverted the removal of the assertion and did the usual nit-picking.  will merge later tonight if it's still open and green, and nobody objects.

Thanks!

**Pull request merged**